exodus_core/analysis/static_analysis.py

```diff
@@ -480,7 +488,7 @@ def print_apk_infos(self):
         print('- App permissions: %s' % len(permissions))
         for p in permissions:
             print('    - %s' % p)
-        print('- App libraries: %s' % len(libraries))
```

Comment:
@Gu1nness any explanation why this was removed ?

Reply:
No. 

I was testing the code and I probably forgot to put it back.